src/Form/AdTechMediaConfigForm.php

```diff
@@ -56,216 +56,260 @@ public function getEditableConfigNames() {
+      '#attributes' => [
+        'class' => ['btn', 'regenerate'],
+      ],
+      '#ajax' => [
+        'callback' => [$this, 'regenerateApiKeyCallback'],
         'event' => 'click',
         'wrapper' => 'edit-api-key',
         'method' => 'replaceWith',
-      ),
-      // '#prefix' => '<i class="mdi mdi-autorenew"></i>',.
-    );
+      ],
+      '#limit_validation_errors' => [],
+    ];
 
-    $form['general']['country'] = array(
+    $form['general']['country'] = [
       '#type' => 'select',
       '#title' => $this->t('Country'),
-      '#options' => array(
+      '#options' => [
         'usa' => $this->t('USA'),
         'md' => $this->t('Moldova'),
-      ),
+      ],
       '#default_value' => $config->get('country'),
-    );
+    ];
 
-    $form['general']['revenue_model'] = array(
+    $form['general']['revenue_model'] = [
       '#type' => 'select',
       '#title' => $this->t('Revenue Model'),
-      '#options' => array(
+      '#options' => [
         'advertising' => $this->t('Advertising'),
         'micropayments' => $this->t('Micropayments'),
-        'advertising_micropayments' => $this->t('Advertising & Micropayments'),
-      ),
+        'advertising+micropayments' => $this->t('Advertising & Micropayments'),
+      ],
       '#default_value' => $config->get('revenue_model'),
       '#description' => $this->t('Select revenue model'),
-    );
+    ];
 
-    $form['general']['email'] = array(
+    $form['general']['email'] = [
       '#type' => 'email',
       '#title' => $this->t('Email Address'),
       '#default_value' => $config->get('email'),
       '#title_display' => 'after',
-    );
+    ];
 
-    $form['general']['activate'] = array(
+    $form['general']['activate'] = [
       '#type' => 'button',
       '#value' => $this->t('Activate'),
-      '#attributes' => array(
-        'class' => array('btn', 'activate'),
-      ),
-    );
+      '#attributes' => [
+        'class' => ['btn', 'activate'],
+      ],
+    ];
 
-    $form['content'] = array(
+    $form['content'] = [
       '#type' => 'details',
       '#title' => $this->t('Content configuration'),
       '#open' => TRUE,
-    );
+    ];
 
-    $form['content']['content_pricing'] = array(
+    $form['content']['content_pricing'] = [
       '#type' => 'textfield',
       '#title' => $this->t('Content pricing'),
       '#default_value' => $config->get('content_pricing'),
       '#description' => $this->t('It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters'),
       '#placeholder' => $this->t('number'),
       '#title_display' => 'after',
-    );
+    ];
 
-    $form['content']['content_currency'] = array(
+    $form['content']['content_currency'] = [
       '#type' => 'select',
       '#title' => $this->t('Content currency'),
-      '#options' => array(
+      '#options' => [
```

Comment:
Please set it up depending on chosen country (@see `/atm-admin/property/supported-countries`)